scss/theme.scss

```
@@ -90,6 +93,18 @@
     }
   }
 
+  .settings-group p {
+    color: lighten($gray, 45%);
+  }
+
+  .dialog-tabs button {
```

```suggestion
  .tab-panels__tab-list li {
```

The classes for this have changed in `master` ☝️

---

Thanks, I also needed to update to use the `is-active` class in 36ee908.